### ISSUE TITLE
Cannot specify valid_odoo_versions as option to pylint from CLI

### ISSUE THREAD

**Issue by katyukha** [OCA/pylint-odoo#175]
This was still available in 1.8.2
In 1.9.3 it is not working

If we call pylint as

```sh
pylint --valid_odoo_versions=$ODOO_VERSION
```

then pylint should check addon version only against specified odoo version or versions

For example, we have addon with version `9.0.0.0.1` and if we run pylint_odoo for it as:

```sh
pylint --valid_odoo_versions=10.0
```

then pylint have to display message about wrong version format.

This will help to control correct addon versions via CI, because without this feature, when some addon is supported on multiple versions of odoo, sometimes it may have version '9.0.0.1.0' on branch '10.0', which is incorrect.

Things that may lead to this incorrect bechavior seems to be [here](https://github.com/OCA/pylint-odoo/blob/b9016b078d21a0d4e56a53d0d8732261279556ce/pylint_odoo/misc.py#L65) - variable `valid_odoo_versions` simply redefined with default version list, making it impossible specify odoo version from pylint commandline.

------

Also this version have one more bug: if addon have not specified version,  then pylint throws error with strange traceback:
```traceback
Traceback (most recent call last):
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/bin/pylint", line 11, in <module>
    sys.exit(run_pylint())
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint/__init__.py", line 13, in run_pylint
    Run(sys.argv[1:])
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint/lint.py", line 1310, in __init__
    linter.check(args)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint/lint.py", line 732, in check
    self._do_check(files_or_modules)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint/lint.py", line 863, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint/lint.py", line 942, in check_astroid_module
    checker.process_tokens(tokens)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint_odoo/checkers/format.py", line 94, in process_tokens
    self.add_message('no-utf8-coding-comment', line=1)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint_odoo/misc.py", line 188, in add_message
    match = self.formatversion(version)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/local/lib/python2.7/site-packages/pylint_odoo/misc.py", line 72, in formatversion
    return re.match(self.config.manifest_version_format_parsed, string)
  File "/home/katyukha/projects/OpenERP/odoo-10.0/venv/lib/python2.7/re.py", line 141, in match
    return _compile(pattern, flags).match(string)
TypeError: expected string or buffer
```

**Comment by moylop260** [comment]
It seems be fixed since that we are using `--valid_odoo_versions=10.0` from https://github.com/OCA/maintainer-quality-tools/blob/580e1dd77014d7e9bd19847312e0ee759aaf482f/travis/run_pylint.py#L74
